style/themes/high-contrast-dark.css

```diff
@@ -918,31 +918,6 @@ input[type="password"]::-webkit-caps-lock-indicator {
   background-color: rgba(0, 64, 64, 0.35);
 }
 
-table.dataTable thead .sorting::after,
```

Comment:
We need to try co increase the contrast on these 2 themes

Can you try this for the dark theme?

```
table.dataTable thead > tr > th.sorting::before {
  opacity: 0.4;
}

table.dataTable thead > tr > th.sorting::after {
  opacity: 0.4;
}

table.dataTable thead > tr > th.sorting:hover::before,
table.dataTable thead > tr > th.sorting:hover::after {
  color: var(--primary-color);
}

table.dataTable thead > tr > th.sorting.sorting_asc::before,
table.dataTable thead > tr > th.sorting.sorting_desc::after {
  opacity: 1;
  color: #fff;
}
```